.azure/obtemplates/git-submit.yml

```diff
@@ -1,24 +1,36 @@
 # Creates the Windows OS build package.
 
+parameters:
+  branch: ''
+  title: ''
+  #msrc: ''
+
 jobs:
 - job: git_push
   displayName: Git push to OS Repo
   pool:
-    vmImage: windows-2019
+    type: windows
   variables:
-  - name: runCodesignValidationInjection
-    value: false
-  - group: VPackDeploymentKeys
+    ob_outputDirectory: $(Build.SourcesDirectory)/artifacts/package
+    ob_sdl_codeSignValidation_excludes: -|**\*.sys;-|**\*.dll;-|**\*.exe
+    runCodesignValidationInjection: false
+    # Update OS Manifest parameters - Settings for checking in manifest to OS repo.
+    ob_updateOSManifest_enabled: true
+    ob_updateOSManifest_osbranch: 'official/rs_onecore_liof1_stack'
+    ob_updateOSManifest_gitcheckinConfigPath: '$(Build.SourcesDirectory)/artifacts/package/GitCheckin.json'
+    ob_updateOSManifest_source: '/minio/netio/quic/msquic/msquic.man'
+    ob_updateOSManifest_destination: '$(Build.SourcesDirectory)/artifacts/package'
```

Comment:
Branch, source and destination are already encoded in the json I believe and are unnecessary to specify here. i didn't need to specify them in my [build.yml](https://mscodehub.visualstudio.com/undocking/_git/undocking?path=/onebranch/v2/build.yml&version=GBmain&line=161&lineEnd=164&lineStartColumn=1&lineEndColumn=73&lineStyle=plain&_a=contents).

Reply:
Oh right. Though, destination will still be needed.